frontend/src/bundles/resume-preview/components/common/description-item/description-item.tsx

```diff
@@ -0,0 +1,17 @@
+import { type DescriptionItemStyles } from '~/bundles/resume-preview/types/types';
+
+interface DescriptionItemProperties {
```

Comment:
Change it's name to just Properties in all places and use type instead of interface.